api/workerapi/workerhttpclient/client.go

```diff
@@ -0,0 +1,98 @@
+package workerhttpclient
```

Comment:
Strongly suggest merging `workerrpcclient` and `workerhttpclient`. Same with merging `workerrpcserver` and `workerhttpserver`.

They are different, but the APIs aren't that big that they need different packages. They are complementing each other, so we won't have any colliding methods that needs them to be in different packages.

Reply:
Merged 👍🏻